Bottom: b38b8cfc05b832cb00b3591c1521509abcf18839
Top:    88a0a806d2a7897d56af99a4394e79a88ec94630
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-13 13:03:03 +0800

Refresh of autofs-5.0.7-add-enable-sloppy-mount-option-to-configure.patch

---

diff --git a/lib/master.c b/lib/master.c
index 836c0fd..eca3523 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1320,12 +1320,6 @@ int master_show_mounts(struct master *master)
 
 		printf("\nMount point: %s\n", ap->path);
 
-		/* Report nulled nulled entries */
-		if (cache_lookup_distinct(master->nc, ap->path)) {
-			printf("\nmap entry %s nulled by previous entry\n",
-			       ap->path);
-		}
-
 		printf("\nsource(s):\n");
 
 		/*
